### ISSUE TITLE
[FEATURE] Chat Format

### ISSUE THREAD

**Issue by earlhsjks** [booksaw/BetterTeams#592]
### What is the feature request for?

Can i format chat?

### Feature requests are not going to be implemented by booksaw, instead you will have to find another developer to make a pull request to implement the feature.

- [X] I understand

**Comment by booksaw** [comment]
Yes, you can edit the syntax of the prefix, see the following line in messages.yml (or your translation file) for more info.
https://github.com/booksaw/BetterTeams/blob/af564e388a0201f67a2b3370a13d3bb32dfa98c7/src/main/resources/messages.yml#L13

**Comment by booksaw** [comment]
This issue is being closed for inactivity. If you are still having problems feel free to open a new issue linking this one, make sure you answer any questions asked in the new issue.
